node-require/add.js

```diff
@@ -0,0 +1,3 @@
+function add(a, b) { return a + b; }
+
+module.exports = { add };
```

Comment:
Interesting. Why do you have {} around the function? I'm surprised this works.

Oh, this is why you have to do `add.add()` and so on. If you simply export the function (without turning it into an object) you can just use `add()` to make the call.